### PR TITLE
Refactor neo.blockchain.node in ES6

### PR DESCRIPTION
**Pull request by hmcq6**

### CODE REVIEW

File: dist/neo.blockchain.node.js [hunk 1]
```diff
@@ -460,100 +396,65 @@ module.exports = function(network) {
...
-      })
-    };
+    this.getRawTransaction = (txid) => new Promise(function (resolve, reject) {
+      node.call({
+        method: 'getrawtransaction',
+        params: [txid,1],
+        id: 0
+      }).then(({ result }) => {
+        resolve(result)
+      }).catch(reject())
```
Comment (hmcq6) [inline dist/neo.blockchain.node.js]: Parenthesis should be removed. The intent was to pass the function reject, not the value

**Comment by rockacola** [comment]
My recommendation is to split "coding style" changes and "refactor" changes into separate tickets as 1 generates a lot of diff entries while the other can be more prompt to bugs.

**Comment by hmcq6** [comment]
@rockacola I broke the work out into two seperate PR's. The code style changes can be found here: https://github.com/CityOfZion/neo-js/pull/22

After that pull request is merged this PR should only require 1 commit (7e0dae938921762a29f1f14a0d9b14a814efec2f), I'll rebase if need be

**Comment by lllwvlvwlll** [comment]
Nice work!

**Pull request merged**